src/common/components/Icon/index.js

```diff
@@ -12,10 +12,17 @@ type IconProps = {
   src?: string,
   button?: boolean,
   children?: any,
+  applyCount?: number,
 };
 
-const Icon = ({ name, size, className, src, button, children, ...props }: IconProps) => {
+const Icon = ({ name, size, className, src, button, children, applyCount, ...props }: IconProps) => {
+  const APPLY_COUNT_THRESHOLD = 1;
   let image: string;
+  let applyBadge: any;
+
+  if (applyCount && applyCount > APPLY_COUNT_THRESHOLD) {
```

Comment:
Initially that was something like this,
and then i taught we might want to add an apply count for any icon, so why not adding it globally.

please checkout the branch and see how it looks like, but i really think it should just be icon property to make sure any need in the future will keep the same logic & css.